.github/workflows/deploy-template.yml

```diff
@@ -49,6 +44,28 @@ jobs:
           aws-secret-access-key: ${{ secrets.aws_secret_access_key }}
           aws-region: "us-gov-west-1"
 
+      - name: Install jq
+        run: sudo apt-get update && sudo apt-get install -y jq
+
+      - name: Verify commit status
+        run: |
+          TOKEN=$(aws ssm get-parameter --name /devops/VA_VSP_BOT_GITHUB_TOKEN --with-decryption --query Parameter.Value --output text)
+
+          STATUS=$(curl -s -H "Authorization: token $TOKEN" \
+            "https://api.github.com/repos/department-of-veterans-affairs/vets-api/commits/${{ inputs.commit_sha }}/status" \
+            | jq -r '.state')
```

Comment:
The only check we care about passing is Code Checks. When I was on support, I saw `Build And Publish Preview Environment` fail a lot, but that one doesn't matter to us.